task-2/src/test/java/MyHashMapTest.java

```diff
@@ -0,0 +1,80 @@
+import com.ury.MyHashMap;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+public class MyHashMapTest {
+    private MyHashMap<String, Integer> map;
+
+    @Test
+    public void testPutAndGet() {
```

Comment:
pls use the following pattern for tests:
test_<method_name>_should_<what are you testing>